lib/coinbase/transaction.rb

```diff
@@ -106,10 +106,17 @@ def raw
     # @return [String] The hex-encoded signed payload
     def sign(key)
       raw.sign(key)
+      @signed = true
 
       raw.hex
     end
 
+    # Returns whether the Transaction has been signed.
+    # @return [Boolean] Whether the Transaction has been signed
+    def signed?
+      @signed
```

Comment:
We could also leverage `Eth::Tx.signed?(raw)` to maintain less state here.

Will need to refactor all of this with multi-protocol support anyways though